src/main/java/com/yelp/nrtsearch/server/grpc/NrtsearchClient.java

```diff
@@ -400,6 +410,28 @@ public List<String> getIndices() {
         .collect(Collectors.toList());
   }
 
+  public void nodeInfo() {
```

Comment:
Do we really need these functions in NrtsearchClient? Can we just to `client.getBlockingStub().<method>` directly in the Command classes? Most of these methods are not reused either.